### DISCUSSION TITLE
除了Smart DNS，还有什么工具能做CNAME？

### DISCUSSION THREAD

**Discussion by 8884361** [coolsnowwolf/lede#11750] [Q&A / 快速问答]
cname /autopatchhk.yuanshen.com/autopatchhk.yuanshen.com.wsdvs.com
cname /autopatchos.starrails.com/autopatchos.starrails.com.wsdvs.com
cname /gst.prod.dl.playstation.net/gst.prod.dl.playstation.net.wtxcdn.com
cname /gs2.ww.prod.dl.playstation.net/gst.prod.dl.playstation.net.wtxcdn.com
cname /zeus.dl.playstation.net/gst.prod.dl.playstation.net.wtxcdn.com
cname /ares.dl.playstation.net/gst.prod.dl.playstation.net.wtxcdn.com

**Comment by ZqinKing** [comment]
adg
